src/openai/resources/beta/threads/threads.py

```diff
@@ -815,6 +842,10 @@ async def create_and_run(
               model associated with the assistant. If not, the model associated with the
               assistant will be used.
 
+          temperature: What sampling temperature to use, between 0 and 2. Higher values like 0.8 will
```

Comment:
This documentation line suggests that the upper bound value is 1 (not 2), since 0.8 is mentioned as "higher value". I think the default is 1, which means 0.8 is still smaller than the default, thus calling it "higher" is IMO mistake.